docs/FAQ.md

```diff
@@ -131,3 +131,90 @@ There is planned support for:
+
+## Will Armory support Bitcoin Gold?
+Yes, but only to the same basic extent as Cash (i.e., users can move their coins to another wallet, preferably ASAP). The same basic directions for Cash apply here, although Gold has no replay attack protections and requires extra vigilance when moving coins.
+
+## Will Armory support Scamcoin, the newest and MOST TOTALLY AMAZINGER BESTEST "airdrop" altcoin that forks off of Bitcoin Core's blockchain?
+Probably not. Never say never but diminishing returns means these coins are the virtual equivalent of [penny stocks](https://www.investopedia.com/terms/p/pennystock.asp), assuming they're not outright scams. It's simply not worth the time of the Armory devs to support the coins, especially if the coin devs can't be bothered to explain their changes to Armory devs. Patches for Armory may be submitted but there's no guarantee they'll be accepted, especially if they alter the GUI or make other fundamental changes to how Armory works.
+
+## Will Armory support Ethereum?
+Probably not, but again, never say never. The answer would depend in large part on the amount of technical changes necessary in order to support the Ethereum network, along with the Ethereum network supporting Armory features whenever possible.
+
+## Can I fork Armory? I want a version that supports an altcoin, adds features not meant for the mainline, etc.
+Sure. Armory is open source. As long as your project follows the appropriate licenses, do what you want! For example, there's a fork that [supports Namecoin](https://github.com/josephbisch/BitcoinArmory/tree/namecoin). Depending on the nature of your project, you're also welcome to submit patches.
+
+## Will Armory add Random Feature X?
+Maybe? Devs work on what they want to work on. Want a feature? Write a patch and submit it.
+
+## What's with the multiple web sites and repositories?
+Armory is now maintained as a community project by goatpig, a former developer at Armory Technologies, Inc. (the business entity that maintained Armory from 2013-2016). He has [a new repo](https://github.com/goatpig/BitcoinArmory) and [web site](https://btcarmory.com/). These are what should be considered the most up-to-date sources for Armory info and materials.
+
+The [original Armory repo](https://github.com/etotheipi/BitcoinArmory) is no longer maintained. It was created by Alan Reiner (Armory's founder) and is now considered abandoned, as Alan moved on in Feb. 2016. The [original web site](https://www.bitcoinarmory.com/) is the intellectual property of ATI and is maintained by principals at ATI. As of the beginning of 2018, the site carries only "major" releases (e.g., 0.96), and not "minor" updates (e.g., 0.96.1) that may contain important bug fixes and new features.
```

Comment:
> As of the beginning of 2018, the site carries only "major" releases (e.g., 0.96), and not "minor" updates (e.g., 0.96.1) that may contain important bug fixes and new features.

That's not necessarily true. It can carry minor updates, but since the current maintainer of that site doesn't really care about it, nothing is actually done in the site. It is open source and also hosted on github pages, so PRs could be made to it to update it.